core/scripts/Voting.js

```diff
@@ -7,13 +7,23 @@ const sendgrid = require("@sendgrid/mail");
 const fetch = require("node-fetch");
 require("dotenv").config();
 const gmailSend = require("gmail-send")();
+const moment = require("moment");
 
 const argv = require("minimist")(process.argv.slice(), { string: ["network"] });
 
 const SUPPORTED_IDENTIFIERS = {
   BTCUSD: {
     dataSource: "CryptoCompare",
     identifiers: { first: "BTC", second: "USD" }
+  },
+  "Custom Index (84.3)": {
```

Comment:
Why not `"Telegram SAFT"`, `"Custom Index (1)"`, and `"Custom Index (100)"`? And why add `"Custom Index (84.3)"`?

Reply:
Done.